packages/algorand-js/src/runtime/runtime.ts

```diff
@@ -318,6 +318,16 @@ export class Runtime {
     }
   }
 
+  // Delete application from account's state and global state
+  deleteApp (accountAddr: string, appId: number): void {
+    if (!this.store.globalApps.has(appId)) {
+      throw new TealError(ERRORS.TEAL.APP_NOT_FOUND);
+    }
+    this.store.globalApps.delete(appId);
+    const account = this.assertAccountDefined(this.store.accounts.get(accountAddr));
```

Comment:
What happens if we delete the `app` and account is not defined?

Reply:
Nice catch